core/scripts/sql/texera_ddl.sql

```diff
@@ -39,3 +40,26 @@ CREATE TABLE IF NOT EXISTS `UserDict` (
     FOREIGN KEY (`userID`) REFERENCES `UserAccount`(`userID`) ON DELETE CASCADE
 )ENGINE=INNODB;
 
+CREATE TABLE IF NOT EXISTS `UserWorkflow` (
+    -- uncomment below to give workflows the concept of ownership
+    -- `userID` INT UNSIGNED NOT NULL,
+    `name` VARCHAR(128) NOT NULL,
+    -- texera workflows use UUID which is 32 chars long
+    `workflowID` VARCHAR (32) NOT NULL,
```

Comment:
VARCHAR(32) might be a problem because there's hyphens in between, like "aaaa-bbbb-cccc". I don't think we need to specify a length, maybe just varchar(max)? performance is not a consideration here. These tables tend to be small. "name" can also be varchar(max)

Reply:
Mysql does not support varhcar(max) and primary key fields has a size limit at 3072 bytes. Therefore, I just changed both of them to a large enough value: 128.